packages/uicore/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.135.0",
+  "version": "3.135.1",
```

Comment:
nit: Shouldn't we be updating the minor version here instead of the patch version?

Reply:
looked like a bugfix to me

Reply:
Ticket originated from lack of contrast and from that perspective, it is a bug. 

But from the perspective of the library, it has been this way since we first implemented and we intended it to be this way. We are only now updating it to have slight more contrast.

That's why I was wondering if it should be minor or patch